template/utils/send_sample_request.js

```diff
@@ -93,15 +93,51 @@ define([
       // send AJAX request, catch success or error callback
       var ajaxRequest = {
           url        : url,
+          dataType   : 'json',
+          contentType: 'application/json',
           headers    : header,
-          data       : param,
           type       : type.toUpperCase(),
           success    : displaySuccess,
           error      : displayError
       };
 
+      if (type === 'get') {
+           ajaxRequest.url = url + paramToQueryParms(param),
+           ajaxRequest.dataType = "text";
+           ajaxRequest.contentType = "text/plain";
+       } else {
+           if ( ! $.isEmptyObject(param)) {
+               ajaxRequest.data = JSON.stringify(param);
+           }
+       }
+
       $.ajax(ajaxRequest);
 
+      function paramToQueryParms(param) {
+        var sendString = [];
+        for (var k in param)
+        {

```

Comment:
The curl brace and for loop should be on the same line. Similar problem to the if/else below.
The coding style should be consistent.